daemon/cmd/daemon_main.go

```diff
@@ -264,6 +264,9 @@ func init() {
 	flags.Bool(option.EnableHealthChecking, defaults.EnableHealthChecking, "Enable connectivity health checking")
 	option.BindEnv(option.EnableHealthChecking)
 
+	flags.Bool(option.EnableHealthCheckNodePort, defaults.EnableHealthCheckNodePort, "flag to allow users to opt-out of the service health server running inside cilium-agent when running NodePort BPF")
```

Comment:
the `flag to allow users to opt-out of the service health server running inside cilium-agent when running NodePort BPF` should be something like `Enables a service health server running inside cilium-agent when running for the NodePort services`